### PR TITLE
10.0 pos mobile patch

### PR DESCRIPTION
**Pull request by Ramil-Mukhametzyanov**

### CODE REVIEW

File: pos_mobile/doc/index.rst [hunk 1]
```diff
@@ -47,3 +47,28 @@ Adding POS icon to a phone homescreen
   * Run the Safari browser on Apple’s iOS and open the POS web page
   * Tap the Share button on the browser’s toolbar
   * Tap the Add to Home Screen icon in the Share menu
+
+
+Fastclick issues
+================
```
Comment (yelizariev) [inline pos_mobile/doc/index.rst]: make it subsection of Installation seciton

File: pos_mobile/doc/index.rst [hunk 2]
```diff
@@ -47,3 +47,28 @@ Adding POS icon to a phone homescreen
...
+
+According to the library note: fastclick offers no benefit on newer browsers, and risks introducing bugs into your application
+
+Possible error is:
+[Violation] A non-passive event listener was added to a scroll lock event . Consider marking the event handler as 'passive' to make the page more responsive. See
```
Comment (yelizariev) [inline pos_mobile/doc/index.rst]: If you want a new line in rendered page, you need ad extra new line

https://github.com/it-projects-llc/pos-addons/blob/ab527b9324dbabfcb9cd3560cf3d5392b5932f55/pos_mobile/doc/index.rst

File: pos_mobile/doc/index.rst [hunk 3]
```diff
@@ -47,3 +47,28 @@ Adding POS icon to a phone homescreen
...
+Fastclick issues
+================
+
+According to the library note: fastclick offers no benefit on newer browsers, and risks introducing bugs into your application
+
+Possible error is:
+[Violation] A non-passive event listener was added to a scroll lock event . Consider marking the event handler as 'passive' to make the page more responsive. See 
+
+The following commit removes fastclick from Odoo:
+https://github.com/odoo/odoo/commit/2f8bfb77d6f9f957448d6267e4037fabf5665a3c
```
Comment (yelizariev) [inline pos_mobile/doc/index.rst]: It's a commit from PR, not the commit that landed to odoo. Correct link: https://github.com/odoo/odoo/commit/d458ec953b179596330091b3efe729868fb56ca2

File: pos_mobile/doc/index.rst [hunk 4]
```diff
@@ -47,3 +47,28 @@ Adding POS icon to a phone homescreen
...
+
+Fastclick issues
+================
+
+According to the library note: fastclick offers no benefit on newer browsers, and risks introducing bugs into your application
+
+Possible error is:
+[Violation] A non-passive event listener was added to a scroll lock event . Consider marking the event handler as 'passive' to make the page more responsive. See 
+
+The following commit removes fastclick from Odoo:
```
Comment (yelizariev) [inline pos_mobile/doc/index.rst]:
```suggestion
The following commit removes fastclick in Odoo 12.0:
```

File: pos_mobile/doc/index.rst [hunk 5]
```diff
@@ -47,3 +47,28 @@ Adding POS icon to a phone homescreen
...
+https://github.com/odoo/odoo/commit/2f8bfb77d6f9f957448d6267e4037fabf5665a3c
+
+To apply them you can use git commands or use patch file remove_fastclick.patch. The patch can be found at module source. Exact commands are as following:
```
Comment (yelizariev) [inline pos_mobile/doc/index.rst]:
```suggestion
To apply the update on earlier odoo version you can use git commands or use patch file remove_fastclick.patch. The patch can be found at module source. Exact commands are as following:
```

**Review by yelizariev — CHANGES_REQUESTED** [review]

**Review by yelizariev — APPROVED** [review]

**Review by ilmir-k — APPROVED** [review]

**Comment by ilmir-k** [comment]
I approve to merge it now

**Review by Ramil-Mukhametzyanov — COMMENTED** [review]
This is the patch in Odoo 10.0 https://github.com/it-projects-llc/pos-addons/issues/728

**Pull request merged**